dune-project

```diff
@@ -1,2 +1,41 @@
-(lang dune 1.0)
+(lang dune 3.16)
```

Comment:
i don't want this, it is rebuild-the-world churn on all extlib users
and no big need for automatic generation of opam file, not like it is going to change much